diagnostics/app/model/diagnostics/css/Css.scala

```diff
@@ -0,0 +1,33 @@
+package model.diagnostics.css
+
+import play.api.libs.json._
+import collection.Map
+
+case class CssReport(
+  href: String,
+  breakpoint: String,
+  className: String,
+  contentType: String,
+  selectors: JsObject ) {
+
+  override def toString(): String = {
+    val selectorMap = selectors.value.asInstanceOf[Map[String, JsBoolean]]
```

Comment:
yes! although I'm sure pattern match/ partial functions use `asInstanceOf` too, however they use it correctly!